src/app/chip_data_model.gni

```diff
@@ -272,8 +272,10 @@ template("chip_data_model") {
           "${_app_root}/clusters/${cluster}/DefaultTimeSyncDelegate.cpp",
           "${_app_root}/clusters/${cluster}/TimeSyncDataProvider.cpp",
         ]
-        defines +=
-            [ "TIME_SYNC_ENABLE_TSC_FEATURE=${time_sync_enable_tsc_feature}" ]
+        defines += [
+          "TIME_SYNC_ENABLE_TSC_FEATURE=${time_sync_enable_tsc_feature}",
+          "TIME_SYNCHRONIZATION_CLUSTER_INCLUDED=1",
```

Comment:
I think ZCL_USING_TIME_SYNCHRONIZATION_CLUSTER_SERVER should work here rather than an additional define. It's in gen_config.h, but it's generated at compile, so you won't find it in the repo.

Reply:
Removed. It was a leftover from a prior attempt.